bmlt-versions.php

```diff
@@ -577,6 +626,27 @@ public function bmltVersionsFunc($atts = [])
                 $releases[12]['date'] = strtotime($yap_date);
                 $releases[12]['version'] = $yap_version;
             }
+
+            if ($workflow) {
+                $workflow_content = '<div class="bmlt_versions_div github">';
+                $workflow_content .= '<ul class="bmlt_versions_ul">';
+                $workflow_content .= '<li class="bmlt_versions_li" id="bmlt-versions-workflow">';
+                $workflow_response = $this->githubLatestReleaseInfo('bmlt-workflow');
+                $workflow_version = $this->githubLatestReleaseVersion($workflow_response);
+                $workflow_date = $this->githubLatestReleaseDate($workflow_response);
+                $workflow_date_ver = $workflow_version . ' (' . date("m-d-Y", strtotime($workflow_date)) . ')';
+                $workflow_content .= '<strong>BMLT Workflow</strong><br>';
+                $workflow_content .= $this->githubReleaseDescription('workflow') . '<br><br>';
```

Comment:
release description should be repo name `bmlt-workflow` instead of just `workflow`